### PR TITLE
Make stdlib's regex flag constants available here.

### PR DESCRIPTION
**Pull request by mattdeboard**
This lets users do like

```python
import rure as re
reo = re.Rure(b"\\p{So}$", flags=re.IGNORECASE)
```

### CODE REVIEW

File: rure/__init__.py [hunk 1]
```diff
@@ -9,8 +11,18 @@
...
 ]
 
+# Make flag constants from stdlib re available here.
+I = IGNORECASE = CASEI
+L = LOCALE
+U = UNICODE
+M = MULTILINE = MULTI
+S = DOTALL = DOTNL
+X = VERBOSE = SPACE
+
```
Comment (davidblewett) [inline rure/__init__.py]: These shortcuts aren't valid; the constant values are different. It's the reason for the `FLAG_MAP` here: https://github.com/davidblewett/rure-python/blob/master/rure/regex.py#L11 .

Comment (davidblewett) [reply rure/__init__.py]: I do like the ergonomics of re-exporting the stdlib constants though.

**Review by davidblewett — CHANGES_REQUESTED** [review]

File: rure/__init__.py (hunk 1 above)
Comment (mattdeboard) [reply rure/__init__.py]: Yeah that's what I was going for was "ergonomics" (good word for it). You can close this if it's too far off the mark.

Comment (davidblewett) [reply rure/__init__.py]: I do think it's worthwhile, but just not setting them equal to the Rure constants. Let's just import the names from `re` and be done.

**Comment by mattdeboard** [comment]
Updated per our convo.

**Pull request merged**